packages/neos-ui-validators/src/Count/index.tsx

```diff
@@ -33,13 +33,9 @@ const Count = (value: any, validatorOptions: CountOptions) => {
 
     const {length} = Object.keys(value);
 
-    if (length === 0) {
```

Comment:
The removal of this check, removes the ability to apply the validation to a optional property. I'd like to keep it as it was, as it works the same for other neos validators. If you need to validate more than one, you can simply add the NotEmptyValidator.

Reply:
Done. 
Thanks for clearing that one up. For me it felt odd at first but I can now understand _why_ this is the way you want it to work.